Mathlib/Data/Int/Init.lean

```diff
@@ -282,6 +282,46 @@ lemma strongRec_of_lt (hn : n < m) : m.strongRec lt ge n = lt n hn := dif_pos _
 
 end strongRec
 
+/-! ### mul -/
+
+-- We want to use these lemmas earlier than the lemmas simp can prove them with
```

Comment:
Question for everyone: Should we consider adding these lemmas to a dedicated simp-set instead of `nolint simpNF`ing them?

Reply:
I think they should be default simp lemmas

Reply:
Ok, let's go with `simp high` in that case.